android/build.gradle

```diff
@@ -56,11 +57,33 @@ android {
 
 }
 
+
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    api "com.revenuecat.purchases:purchases:$purchases_version"
-    
+    implementation 'androidx.annotation:annotation:1.2.0'
+
+    // TODO: un-comment the api line and remove the implementation once purchases-android is released
+    // api "com.revenuecat.purchases:purchases:$purchases_version"
+    // api "com.revenuecat.purchases:purchases:4.0.5"
+    implementation('com.revenuecat.purchases:purchases') {
+        version {
+            branch = 'develop'
+        }
+    }
+
+    implementation('com.revenuecat.purchases:common') {
+        version {
+            branch = 'develop'
+        }
+    }
+
+    implementation('com.revenuecat.purchases:utils') {
+        version {
+            branch = 'develop'
+        }
+    }
```

Comment:
maybe there's a nicer way to group these? I don't know how flexible Gradle is in terms of loops or lists

Reply:
i'm realizing how much i need to brush up on gradle..